apollo-router/src/plugins/cache/entity.rs

```diff
@@ -1359,9 +1359,10 @@ fn extract_cache_keys(
     Ok(res)
 }
 
-pub(crate) fn hash_entity_key(representation: &Value) -> String {
+pub(crate) fn hash_entity_key(representation: &mut Value) -> String {
```

Comment:
Same comment as the other `&mut` site ^^
```suggestion
/// Returns a hash for the given representation, independent of field order. Destructively sorts any objects in the input.
pub(crate) fn hash_entity_key(representation: &mut Value) -> String {
```